### ISSUE TITLE
[Feature Request] Multiple sidebar with deeply nested subfolder

### ISSUE THREAD

**Issue by mrleblanc101** [jooy2/vitepress-sidebar#174]
Hi,
I don't think this is currently possible.
But our documentation is very extensive and has many folders and subfolder.
To make it easier to navigate, it would be nice if we could have a sidebar for each subfolder instead of collapsible sections.
Here is an exemple:
<img width="320" alt="Screenshot 2024-09-12 at 11 55 38 PM" src="https://github.com/user-attachments/assets/80a1cebd-e149-47c3-b693-1cd3b4b692a5">

**Comment by jooy2** [comment]
Hello! Thank you for using `vitepress-sidebar`.

This plugin implements functionality based on the limits provided by VitePress.

If you want to simplify your menu display by showing different sidebar menus based on specific paths (URIs), i recommend learning about **Multiple Sidebar** feature.

This is explained in detail in the documentation below for you to implement in your own plugin:
https://vitepress-sidebar.cdget.com/advanced-usage/multiple-sidebars-how-to

If you're still having trouble, i'd appreciate a more detailed description of your requirements.

Regards,

**Comment by mrleblanc101** [comment]
@jooy2 Yes, I already use this feature, but I need to manually add every directory to scan, and this won't add the directories inside the directories (reacusively).

**Comment by mrleblanc101** [comment]
Here is a a visual exemple of what I want:
![2024-09-16 08 08 02](https://github.com/user-attachments/assets/6eb8c0ab-36a9-4ef6-b6ea-9ef4633c272b)
